### DISCUSSION TITLE
how can I undo selection ?

### DISCUSSION THREAD

**Discussion by zhaob1n** [helix-editor/helix#5633] [Q&A]
When selecting text, I often use `x` many times to select multilines. But how can I undo the last line selection if I select more lines than I want? I also use other commands such as `t` and `f` to extend selection and sometimes wanna undo it. I think there should be a specific command for undoing the last selection behavior.

**Comment by Proful** [comment]
https://github.com/helix-editor/helix/discussions/5548

**Comment by the-mikedavis** [comment]
https://github.com/helix-editor/helix/issues/1596
